src/components/ButtonGroup/GroupButton/index.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import GroupButton from '.';
+import { noop } from '../../../util';
```

Comment:
[should] this should be an absolute import.

Reply:
fixed.